### ISSUE TITLE
Frontend starting a session, when it shouldn't.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#6196]
The Bolt frontend still starts a session (in Bolt 3.2.4). Fresh setup, no extensions.. 

![screen shot 2016-12-22 at 13 54 46](https://cloud.githubusercontent.com/assets/1833361/21426346/b648f516-c84e-11e6-8bd4-c6342d522e37.png)

Pinging @CarsonF Could you look into this? I thought we fixed this.

**Comment by maetthu** [comment]
(as discussed with @bobdenotter on Slack)

Within my environment, I can reproduce it with debug-mode (it's not publicly accessible, sorry). If I set ```debug: true```, a session is started

```
HTTP/1.1 200 OK
Cache-Control: private
Connection: keep-alive
Content-Encoding: gzip
Content-Type: text/html; charset=UTF-8
Date: Thu, 22 Dec 2016 13:40:53 GMT
Server: nginx/1.10.1
Set-Cookie: bolt_session_X=Y; expires=Thu, 05-Jan-2017 13:40:53 GMT; Max-Age=1209600; path=/; domain=XXX; HttpOnly
Strict-Transport-Security: max-age=3600
Transfer-Encoding: chunked
X-Debug-Token: ef988a
```

, with ```debug: false```, it is not:

```
HTTP/1.1 200 OK
Cache-Control: public, s-maxage=3600
Connection: keep-alive
Content-Encoding: gzip
Content-Type: text/html; charset=UTF-8
Date: Thu, 22 Dec 2016 13:42:32 GMT
Server: nginx/1.10.1
Strict-Transport-Security: max-age=3600
Transfer-Encoding: chunked
```

A deployment run does clear out the cache folder though, but sessions are configured to use a redis host instead of local storage, so previously started sessions are still available. 

I don't have any publicly available extensions installed, but I do have some local ones - none of which are accessing session data, or at least not directly.

**Comment by GawainLynch** [comment]
[Thumbnail controller abusing sessions](https://github.com/bolt/bolt-thumbs/blob/master/src/Controller.php#L150)

**Comment by GawainLynch** [comment]
Fixed in https://github.com/bolt/bolt-thumbs/pull/35

**Comment by bobdenotter** [comment]
Fixed in bolt/bolt-thumbs#35
